tensorflow_probability/python/experimental/mcmc/covariance_reducer.py

```diff
@@ -52,6 +52,33 @@ class CovarianceReducer(reducer_base.Reducer):
   `CovarianceReducer` is meant to fit into the larger Streaming MCMC
   framework. `RunningCovariance` in `tfp.experimental.stats` is better suited
   for more generic streaming covariance needs.
+
+  There are two ways to stream over MCMC samples. The first is to manually
+  wrap `CovarianceReducer` in a `tfp.experimental.mcmc.WithReductions`
+  `TransitionKernel`. Doing so enables a wide variety of possible compositions.
+  For example, to perform covariance calculation on samples that survive
+  thinning and burn-in, `WithReductions` should wrap around an appropriate
+  `SampleDiscardingKernel`.
+
+  If the sole objective is to estimate covariance, it may be more convenient to
+  use a pre-defined driver like `tfp.experimental.mcmc.sample_fold`.
+  `sample_fold` will automatically create the Transition Kernel onion, apply
+  `kernel` samples to update reducer states, and `finalize` calculations.
+
+  ```
+  python
+
+  kernel = ...
+  reducer = tfp.experimental.mcmc.CovarianceReducer()
+  covariance_estimate = tfp.experimental.mcmc.sample_fold(
```

Comment:
This isn't right, sample_fold returns 3 outputs.

Reply:
Ah good catch! This was a fix we made in the last round of reviews and I forgot to change this